### PR TITLE
Create Makefile to support Ledger

### PR DESCRIPTION
**Pull request by dylanschultzie**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  ldflags += -linkmode=external -extldflags "-Wl,-z,muldefs -static"
+endif
+ldflags += $(LDFLAGS)
+ldflags := $(strip $(ldflags))
+
+BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)'
+
+#### Command List ####
+
+all: lint install
```
Comment (dennyX7) [inline Makefile:61]: There is no rule for "lint". Add the rule for lint.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+build_tags := $(strip $(build_tags))
+
+whitespace :=
+whitespace += $(whitespace)
+comma := ,
+build_tags_comma_sep := $(subst $(whitespace),$(comma),$(build_tags))
+
+# process linker flags
+
+ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=idep \
```
Comment (dennyX7) [inline Makefile:45]: Modify the values to,

version.Name=ion
version.ServerName=iond
version.Version=v1.0.0

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		  -X github.com/cosmos/cosmos-sdk/version.ServerName=iond \
```
Comment (dennyX7) [inline Makefile:46]: Instead of `github.com/cosmos/cosmos-sdk/version.ServerName=iond` use `github.com/cosmos/cosmos-sdk/version.AppName=iond`

**Comment by dylanschultzie** [comment]
@dennyX7 apologies for the delay here, updated 👍

**Pull request closed without merging**